### ISSUE TITLE
Error while retrieving text from pdf with an empty page

### ISSUE THREAD

**Issue by erikliberal** [LibrePDF/OpenPDF#34]
While reading text per page of a Pdf had had some issues when it had a blank page. Any other contents were loaded just fine
```
java.lang.NullPointerException at com.lowagie.text.pdf.parser.PdfTextExtractor.getContentBytesFromContentObject(PdfTextExtractor.java:157)
  at com.lowagie.text.pdf.parser.PdfTextExtractor.getContentBytesForPage(PdfTextExtractor.java:138)
  at com.lowagie.text.pdf.parser.PdfTextExtractor.getTextFromPage(PdfTextExtractor.java:223)
  at com.lowagie.text.pdf.parser.PdfTextExtractor.getTextFromPage(PdfTextExtractor.java:199)
```

**Comment by andreasrosdal** [comment]
Thanks for reporting this.
Perhaps you can submit a pull-request to fix this?

Which version of OpenPDF are you using?

**Comment by andreasrosdal** [comment]
This is the relevant code: 
https://github.com/LibrePDF/OpenPDF/blob/master/openpdf/src/main/java/com/lowagie/text/pdf/parser/PdfTextExtractor.java#L157

**Comment by erikliberal** [comment]
Yeah, after I'm done with some tasks I'm currently working on that's what I'll do

I'm using version 1.0.1

**Comment by kmix** [comment]
This appears to be fixed and can likely be closed. Unable to reproduce in local test environment.

https://github.com/LibrePDF/OpenPDF/commit/6dd8db39c021ec25ce4c60c729b77aad1f43632d

**Comment by asturio** [comment]
@erikliberal Should work with 1.0.3+. Can you check that?

**Comment by daviddurand** [comment]
This problem was fixed in **1.0.3** by
https://github.com/LibrePDF/OpenPDF/commit/6dd8db39c021ec25ce4c60c729b77aad1f43632d

**Comment by asturio** [comment]
I'm closing this now. If the issue isn't fixed someone will open it again.
